src/views/QuizView/quizView.js

```diff
@@ -0,0 +1,230 @@
+  return rightArrow;
+}
+
+function createLeftArrow() {
+  const leftArrow = document.createElement('div');
+  leftArrow.setAttribute('id', 'navigation-arrows-left');
+  const leftArrowImage = document.createElement('img');
+  leftArrowImage.src = './leftarrow.png';
+  leftArrow.appendChild(leftArrowImage);
+  leftArrowImage.addEventListener('click', previousQuestion);
+  return leftArrow;
+}
+
+function createLifeline() {
+  const lifelineContainer = document.createElement('div');
+  lifelineContainer.setAttribute('id', 'lifeline');
+  const lifelineImage = document.createElement('img');
+  lifelineImage.src = './lifering.png';
+  lifelineContainer.appendChild(lifelineImage);
+  lifelineImage.addEventListener('click', useLifeline);
+  return lifelineContainer;
+}
+
+function createMenuButton() {
+  const menuButton = Button('MENU', 'quizViewMenuButton', null, 'click', navigateToMenu);
+  return menuButton;
+}
+
+function navigateToMenu() {
+  window.location.hash = '';
+}
+
+function createLayout() {
+  const container = document.createElement('div');
+  container.setAttribute('id', 'quizView');
+
+  const header = document.createElement('h1');
+  header.innerText = 'LIVE QUIZ';
+
+  const image = document.createElement('img');
+
+  const question = document.createElement('div');
+  question.setAttribute('id', 'question-text');
+
+  const answers = document.createElement('div');
+  answers.setAttribute('id', 'answers');
+
+  container.append(
+    header,
+    createLifeline(),
+    createQuestionNumbers(),
+    image,
+    createTimer(),
+    question,
+    createRightArrow(),
+    createLeftArrow(),
+    answers,
+    createMenuButton(),
+  );
+
+  return container;
+}
+
+function renderQuizData() {
+  if (current === QuizSettings.questionsNum) {
+    renderChoiceModal();
+  }
+  current = current < 0 ? 0 : current;
+  current = current === QuizSettings.questionsNum ? current - 1 : current;
+  const numbers = document.getElementById('question-numbers').children;
+  numbers.item(current).setAttribute('id', 'current-question');
+  const image = document.querySelector('#quizView > img');
+  image.src = userAnswers[current].question.imageUrl;
+
+  const question = document.getElementById('question-text');
+  question.innerText = userAnswers[current].question.question.toUpperCase();
+
+  const answersContainer = document.getElementById('answers');
+  const answers = userAnswers[current].question.getAnswers();
+
+  for (let i = 0; i < answers.length; i++) {
+    const answerBtn = Button(answers[i], 'answer', false, 'click', nextQuestion);
+    if (answers[i].toUpperCase() === userAnswers[current].answer) {
+      answerBtn.classList.add('selected');
+    }
+    answersContainer.appendChild(answerBtn);
+  }
+  startTime = getTime();
+}
+
+function getTime() {
+  timerMinutes = document.getElementById('timer-minutes').innerText;
+  timerSeconds = document.getElementById('timer-seconds').innerText;
```

Comment:
I do not like that the state(minutes and seconds) is shared through HTML element 😞